primitives/api/src/lib.rs

```diff
@@ -154,41 +154,11 @@ pub use sp_api_proc_macro::decl_runtime_apis;
 ///
 /// # Example
 ///
-/// ```rust
+/// ```ignore
```

Comment:
That is something I'm totally against and will not approve. Breaking tests for releasing is no good idea. The cyclic dependency you are mentioning is only a `dev-dependency`. We should just comment them out when we do a release.

You could also try to implement `Block` and `Header` for some fake types.